packages/vite-plugin-checker/src/checkers/vueTsc/languagePlugins.cjs

```diff
@@ -0,0 +1,45 @@
+const path = require('path')
+const { removeEmitGlobalTypes } = require('vue-tsc')
+
+const vueTscDir = path.dirname(require.resolve('vue-tsc/package.json'))
+const vue = /** @type {typeof import('@vue/language-core')} */ (
+  require(require.resolve('@vue/language-core', { paths: [vueTscDir] }))
```

Comment:
`@vue/language-core` is from the dependency `vue-ts`. So in a `hoist=false` enabled pnpm project. We can't reach `@vue/language-core` from `vite-plugin-checker` as it's a phantom dependency. I'd not prefer to require user to install `@vue/language-core` to dependencies again.

So based on current implementation, I think it's better to resolve `vue-tsc` first, then resolve `@vue/language-core` afterward.

We could fix this in the next PR.

Reply:
Oh, it's already in the current implementation. I made this mistake comment by just seeing the dev dep of `@vue/language-core`.